Makefile

```diff
@@ -484,7 +484,8 @@ set-manifest-image:
 ## --------------------------------------
 
 ## latest git tag for the commit, e.g., v0.3.10
-RELEASE_TAG := $(shell git describe --abbrev=0 2>/dev/null)
+RELEASE_TAG ?= $(shell git describe --abbrev=0 2>/dev/null)
+PREVIOUS_TAG ?= $(shell git describe --abbrev=0 $(RELEASE_TAG)^ 2>/dev/null)
```

Comment:
Would it be possible to rely on the published releases in github and to exclude pre-release versions for these?

Reply:
yeah let me try to use a github action to get it

Reply:
so I added https://github.com/kubernetes-sigs/cluster-api/pull/5188/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R27 which will get the latest non-draft, non-pre-release release from the repo, and set that as PREVIOUS_TAG in env (since this has ?= it will only be set if empty). The problem with this is that if we're dealing with two minor releases like we often are, the "latest" release won't always be the correct previous release. And if we're releasing a .0, we'd want to compare with the previous minor. I'm not sure how to best correctly select the previous ref in that case.

Reply:
Something we can do is to reuse some of the code we have for the book to get the latest release based on the series from go modules https://github.com/kubernetes-sigs/cluster-api/blob/master/hack/tools/mdbook/releaselink/releaselink.go#L56-L94

Reply:
/hold

Reply:
/hold cancel